store/tikv/2pc.go

```diff
@@ -734,3 +752,26 @@ func appendBatchBySize(b []batchKeys, region RegionVerID, keys [][]byte, sizeFn
 	}
 	return b
 }
+
+// batchContains returns true if batch contains the key
+func batchContains(batch batchKeys, key []byte) bool {
+	for _, k := range batch.keys {
+		if bytes.Compare(k, key) == 0 {
+			return true
+		}
+	}
+	return false
+}
+
+// splitBatches split the batches to primary and others according to the primaryKey
+func splitBatches(batches []batchKeys, primaryKey []byte) (primary batchKeys, others []batchKeys) {
```

Comment:
I think you should add unit tests for `splitBatches` and `batchContains`.

Reply:
What happen if `primaryKey` is not in any of the `[]batchKeys`? 
@shafreeck